README.md

```diff
@@ -37,7 +37,7 @@ module "aks" {
   client_secret                    = "your-service-principal-client-password"
   kubernetes_version               = "1.19.3"
   orchestrator_version             = "1.19.3"
-  prefix                           = "prefix"
```

Comment:
Then `prefix` will still be required for `dns_prefix`

Reply:
Ok, I revert this change.